api/src/utils/get-cache-key.ts

```diff
@@ -10,7 +10,7 @@ export function getCacheKey(req: Request): string {
 	const info = {
 		user: req.accountability?.user || null,
 		path,
-		query: isGraphQl ? pick(req.query, ['query', 'variables']) : req.sanitizedQuery,
+		query: isGraphQl ? pick(req.body, ['query', 'variables']) : req.sanitizedQuery,
```

Comment:
This will fail on get requests I think. It needs to be query or body based on get vs post respectively

Reply:
Fixed 👍